src/Collection/Hero.php

```diff
@@ -293,6 +327,7 @@ public static function format_background( $value, $post_id, $field ) {
 
 			if ( 'video' === $item['type'] ) {
 				$new_value[ $index ]['video'] = Utils::get_video_embed_url( $item['video'] );
```

Comment:
I see. Maybe we can add an `if` in order to check if  the type is an url just in case? I don't think is big deal, but it would be safer right? 
Let me now your thoughts! @adamf321  

Thanks